vip-helpers/vip-utils.php

```diff
@@ -495,28 +495,19 @@ function wpcom_vip_file_get_contents( $url, $timeout = 3, $cache_time = 900, $ex
 	$extra_args = wp_parse_args( $extra_args, $extra_args_defaults );
 
 	$cache_key       = md5( serialize( array_merge( $extra_args, array( 'url' => $url ) ) ) );
-	$backup_key      = $cache_key . '_backup';
-	$disable_get_key = $cache_key . '_disable';
+	$backup_key      = md5( $cache_key . '_backup' );
+	$disable_get_key = md5( $cache_key . '_disable' );
```

Comment:
Not totally clear why we need to md5 an already mostly md5 key? Plus, this makes debugging harder if we ever need to look up keys.